config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
And docs around the need for this setting vs. `root.path` would be nice (cherry-pick from [here][1]?).  I'm backing off of my earlier [suggestion to put all of this information in `root.path`][2], because it would be awkward to combine a POSIX path-to-image with a Windows path-to-root-inside-the-image (if you were launching a Windows VM on from a POSIX system).

[1]: https://github.com/opencontainers/runtime-spec/pull/405#discussion_r61358293
[2]: https://github.com/opencontainers/runtime-spec/pull/405#discussion_r61373485